repo2docker/docker.py

```diff
@@ -69,6 +74,14 @@ class DockerEngine(ContainerEngine):
         config=True,
     )
 
+    extra_buildx_build_args = List(
+        [],
```

Comment:
If you want to specify the type of items in the list, give a single TraitType instance here (not list):
```suggestion
        Unicode(),
```

Reply:
Done!